cdisc_rules_engine/rules_engine.py

```diff
@@ -134,9 +136,18 @@ def validate_single_rule(
                 is_split_dataset(datasets, dataset_domain),
                 datasets,
             ):
-                result: List[Union[dict, str]] = self.validate_rule(
-                    rule, dataset_path, datasets, dataset_domain
-                )
+                result = None
+                if vx in ("Y", "YES"):
```

Comment:
It seems like these are just trying to bubble up exceptions that occur during xml validation. I think they should be logged and handled in the reader so we don't have this check for vx every time we have to get define metadata.